featurebyte/query_graph/node/generic.py

```diff
@@ -924,6 +930,12 @@ def _derive_sdk_code(
         return statements, out_var_name
 
 
+class LookupTargetNode(BaseLookupNode):
+    """LookupTargetNode class"""
+
+    type: Literal[NodeType.LOOKUP_TARGET] = Field(NodeType.LOOKUP_TARGET, const=True)
+
```

Comment:
Curious what is the difference between `LookupNode` and `LookupTargetNode`? Could we reuse the same node if there is no difference?

Reply:
there'll be some different parameters, and also some different handling, which requires a different node!

Reply:
Could we update existing lookup related tests to parametrize it to handle different lookup node type?

Reply:
yeap! will update the tests accordingly :)